src/components/AppNavigation/CalendarListItemSharing.vue

```diff
@@ -105,14 +113,40 @@ export default {
 						})
 					}
 					return list
-				}, [])
+				}, []) 
+
+				let circles = resultCircles.data.ocs.data.circles
```

Comment:
All this circles stuff needs to move inside the `.then()` block. 
And you will have to push stuff into the list array.